phone_book_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+func TestCreatePhoneBook(t *testing.T) {
```

Comment:
Tests are incomplete. Where is a test body containing `createPhoneBook` call and output comparison?